scripts/export_large_tsv/README.md

```diff
@@ -0,0 +1,8 @@
+## Export (download) a tsv file for a given entity in a workspace
+This script generates a local tsv file for a given entity type when the number of individual entities is too large to download from the GUI.
```

Comment:
Is it just the given entity type? It's the specific (named) entity, right?

Reply:
entity_type like "sample" or "participant" and it will export the entities of that given entity_type. I should just add that example into the readme!